src/remoted/manager.c

```diff
@@ -178,7 +178,7 @@ void save_controlmsg(const keyentry * key, char *r_msg, size_t msg_length, int *
             w_mutex_unlock(&lastmsg_mutex);
             agent_id = atoi(key->id);
 
-            if (OS_SUCCESS != wdb_update_agent_keepalive(agent_id, logr.worker_node?"syncreq":"synced", wdb_sock)) {
+            if (OS_SUCCESS != wdb_update_agent_keepalive(agent_id, AGENT_CS_PENDING, logr.worker_node?"syncreq":"synced", wdb_sock)) {
                 mwarn("Unable to set last keepalive as pending");
```

Comment:
This warning message might be more clear, like "Unable to set status as pending" or "Unable to save last keepalive"

Reply:
Fixed